templates/jenkins.j2

```diff
@@ -0,0 +1,33 @@
+NAME={{ jenkins_name }}
+JAVA={{ jenkins_java_location }}
+JAVA_ARGS="{{ (jenkins_java_opts + jenkins_java_user_opts) | join(" ") }}"
+PIDFILE={{ jenkins_pidfile_path }}
+JENKINS_USER=$NAME
+JENKINS_GROUP=$NAME
+JENKINS_WAR={{ jenkins_war_path }}
+JENKINS_HOME={{ jenkins_home_path }}
+
+RUN_STANDALONE={{ jenkins_run_standalone }}
+
+JENKINS_LOG={{ jenkins_log }}
+MAXOPENFILES=8192
+{% if jenkins_set_umask %}
+UMASK={{ jenkins_umask }}
+{% endif %}
+
+HTTP_PORT={{ jenkins_http_port }}
+
+
+PREFIX=/$NAME
+
+# arguments to pass to jenkins.
+# --javahome=$JAVA_HOME
+# --httpPort=$HTTP_PORT (default 8080; disable with -1)
+# --httpsPort=$HTTP_PORT
+# --ajp13Port=$AJP_PORT
+# --argumentsRealm.passwd.$ADMIN_USER=[password]
+# --argumentsRealm.$ADMIN_USER=admin
+# --webroot=~/.jenkins/war
+JENKINS_ARGS="{{ (jenkins_args + jenkins_user_args)| join(" ") }}"
```

Comment:
Similar comment as the one above for `jenkins_java_opts`. I don't think `jenkins_user_args` is necessary. The user can override the entire list for `jenkins_args`.